flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -301,10 +301,16 @@ public boolean databaseExists(String databaseName) throws CatalogException {
 	}
 
 	@Override
-	public void dropDatabase(String name, boolean ignoreIfNotExists) throws DatabaseNotExistException,
-			DatabaseNotEmptyException, CatalogException {
+	public void dropDatabase(String name, boolean ignoreIfNotExists)
+			throws DatabaseNotExistException, DatabaseNotEmptyException, CatalogException {
+		dropDatabase(name, ignoreIfNotExists, true);
+	}
+
+	@Override
+	public void dropDatabase(String name, boolean ignoreIfNotExists, boolean isRestrict)
```

Comment:
rename the param to `restrict`? to be consistent with param `cascade` in the `public void dropDatabase(String name, boolean deleteData, boolean ignoreIfNotExists, boolean cascade)`